fdbclient/CoordinationInterface.h

```diff
@@ -59,22 +59,34 @@ struct ClientLeaderRegInterface {
 class ClusterConnectionString {
 public:
 	ClusterConnectionString() {}
-	ClusterConnectionString(std::string const& connectionString);
-	ClusterConnectionString(std::vector<NetworkAddress>, Key);
-
-	std::vector<NetworkAddress> const& coordinators() const { return coord; }
+	ClusterConnectionString(std::string const& connStr);
+	ClusterConnectionString(std::vector<NetworkAddress> coordinators, Key key);
```

Comment:
`const std::vector<NetworkAddress>&` if possible

Reply:
I see sometimes flow automatically compiles params to const references in .g.cpp, when does that happen?

Reply:
ACTOR function converts all arguments into const&.